README.md

```diff
@@ -41,169 +39,191 @@ General documentation surrounding workers development and using `wrangler` can b
 
 - ### 👯 `generate`
 
-    Scaffold a project, including boilerplate for a Rust library and a Cloudflare Worker.
-    You can pass a name and template to this command optionally.
+  Scaffold a project, including boilerplate for a Rust library and a Cloudflare Worker.
+  You can pass a name and template to this command optionally.
 
-    ```bash
-    wrangler generate <name> <template> --type=["webpack", "javascript", "rust"]
-    ```
+  ```bash
+  wrangler generate <name> <template> --type=["webpack", "javascript", "rust"]
+  ```
 
-    All of the arguments and flags to this command are optional:
-        - `name`: defaults to `worker`
-        - `template`: defaults to the [`https://github.com/cloudflare/worker-template`](https://github.com/cloudflare/worker-template)
-        - `type`: defaults to "webpack"
+  All of the arguments and flags to this command are optional: - `name`: defaults to `worker` - `template`: defaults to the [`https://github.com/cloudflare/worker-template`](https://github.com/cloudflare/worker-template) - `type`: defaults to "webpack"
```

Comment:
looks like we lost the list here. hard to tell if this is from the merge (likely no) or something in a branch. dealers choice as to how to fix.
```suggestion
  All of the arguments and flags to this command are optional: 
  
  - `name`: defaults to `worker`
  - `template`: defaults to the [`https://github.com/cloudflare/worker-template`](https://github.com/cloudflare/worker-template)
  - `type`: defaults to "webpack"
```